pkgs/misc/emulators/wine/base.nix

```diff
@@ -62,6 +62,8 @@ stdenv.mkDerivation ((lib.optionalAttrs (buildScript != null) {
   ++ lib.optionals stdenv.isDarwin (with pkgs.buildPackages.darwin.apple_sdk.frameworks; [
      CoreServices Foundation ForceFeedback AppKit OpenGL IOKit DiskArbitration Security
      ApplicationServices AudioToolbox CoreAudio AudioUnit CoreMIDI OpenAL OpenCL Cocoa Carbon
+     # Needed for CFNotificationCenterAddObserver symbols.
+	   pkgs.buildPackages.darwin.cf-private
```

Comment:
This should not be needed anymore: #63381

Reply:
Could you please revert this two lines?